packages/react-core/src/components/Button/Button.tsx

```diff
@@ -20,6 +21,16 @@ export enum ButtonType {
   submit = 'submit',
   reset = 'reset'
 }
+
+export interface BadgeCountObject {
```

Comment:
Sorry I missed this with the last review.  In order for the `BadgeCountObject` to show up in the props table at the bottom of the documentation page, we need to add it to the `propComponents` `Button.md` file to the. The change is needed in line 5.
It would be like this: `propComponents: ['Button', 'BadgeCountObject' ]`